SYCL/BFloat16/bfloat16_builtins.cpp

```diff
@@ -0,0 +1,241 @@
+// REQUIRES: cuda
```

Comment:
Now that it uses aspects, is this still needed?

Reply:
I'm not actually sure. Maybe it isn't needed. Probably I can only find out by removing it and running the CI to test all backends. I'll push a commit removing it and see what happens.

Reply:
It appears that it fails to compile for other backends. Let us reintroduce this requirement. Could you please make a note specifying that it currently fails for other backends?